src/Jawa/Event/MediaType.elm

```diff
@@ -0,0 +1,34 @@
+module Jawa.Event.MediaType exposing (MediaType, decoder, encoder)
+
+{-|
+
+@docs MediaType, decoder, encoder
+
+-}
+
+import Json.Decode
+import Json.Encode
+
+
+{-| This event is not documented.
+-}
+type alias MediaType =
+    { mediaType : String
```

Comment:
I'm assuming this is an enum of some kind, but I don't know what the possible values are. Seems likely that it's either `"audio"` or `"video"`.

https://github.com/jwplayer/jwplayer/blob/c280de15ff78927c026010e4e7a83054e4638240/src/js/providers/default.ts#L33

Reply:
Resolved by 61eb91a2c1fcdd7ca3620adf9f63b18b9798f45a.